tests/openfhe/end_to_end/BUILD

```diff
@@ -0,0 +1,22 @@
+# See README.md for setup required to run these tests
+
+load("//bazel:lit.bzl", "glob_lit_tests")
+
+package(
+    default_applicable_licenses = ["@heir//:license"],
+    default_visibility = ["//visibility:public"],
+)
+
+glob_lit_tests(
+    name = "all_tests",
+    data = [
+        "src/main.cc",
+        "@heir//tests:test_utilities",
+    ],
+    default_tags = [
+        "manual",
```

Comment:
ah i think `src/main.cc` isn't a thing yet - since you haven't added the full e2e test execution, can you remove the `manual` and data here so that when the e2e test functionality is added we can review that part there? (Since these don't depend on cargo, i assume we don't need to run them with manual)